matrix/mat64/pool.go

```diff
@@ -81,6 +87,12 @@ func init() {
 				Data: make([]float64, l),
 			}}
 		}
+		poolFloats[i].New = func() interface{} {
+			return make([]float64, l)
+		}
+		poolInts[i].New = func() interface{} {
+			return make([]int, l)
```

Comment:
Yeah. I am on ubuntu which has generally good fonts for this, so I don't notice.